pkgs/applications/networking/instant-messengers/pidgin-plugins/purple-lurch/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, pidgin, minixml, libxml2, sqlite, libgcrypt, cmake }:
+
+stdenv.mkDerivation rec {
+  name = "purple-lurch-0.6.7";
+  src = fetchFromGitHub {
+    owner = "gkdr";
+    repo = "lurch";
+    rev = "3156e14c2dcf9968438aceb7fda8f82eae6f552d";
```

Comment:
Since is a release on upstream github repo, just do "v${version}" instead of specifying rev manually.